js/languages/en-US.json

```diff
@@ -71,12 +71,13 @@
   "Welcome": "Welcome",
   "CreateStore": "Become a Store",
   "GoToMyPage": "My Page",
-  "ShowOnlyFollowerListings": "Filtered Listings",
+  "Random": "Random",
+  "ShowOnlyFollowerListings": "From Friends",
```

Comment:
Should this say 'friends'? We don't use the world friends for people you follow anywhere else.